### PR TITLE
portalicious: payment chart

### PR DESCRIPTION
**Pull request by aberonni**
[AB#31325](https://dev.azure.com/redcrossnl/7d17f2a9-3f18-40f1-8c29-98568fbca7e4/_workitems/edit/31325)

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] I have added tests wherever relevant
- [x] I have made sure that all automated checks pass before requesting a review
- [x] I do not need any deviation from our PR guidelines

### CODE REVIEW

File: interfaces/Portalicious/src/app/utils/chart.ts [hunk 1]
```diff
@@ -4,6 +4,9 @@ import ChartDataLabels from 'chartjs-plugin-datalabels';
 export function registerChartDefaults() {
   const documentStyle = getComputedStyle(document.documentElement);
 
+  Chart.defaults.font.family = 'Montserrat, sans-serif';
```
Comment (elwinschmitz) [inline interfaces/Portalicious/src/app/utils/chart.ts:7]: There is no magical `var(--font-display)` Tailwind-utility, right?

Or like with the `--text-color` below?

Using the TailwindConfig as-is wouldn't really work unfortunately... :/

Comment (aberonni) [reply interfaces/Portalicious/src/app/utils/chart.ts:7]: No, I tried to pull it out but couldn't find a way to make it happen.

Worst case scenario the fonts in the chart are slightly off - feels like a fair risk to me given that we literally have one chart on the portal (for now).

File: interfaces/Portalicious/src/app/pages/project-payment/components/project-payment-chart/project-payment-chart.component.html [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<p-chart
+  type="bar"
+  [data]="chartData()"
+  [options]="chartOptions"
+  responsive
+  class="[&>div]:max-h-96"
+/>
```
[the remark this answers is not in the document]
Comment (aberonni) [reply interfaces/Portalicious/src/app/pages/project-payment/components/project-payment-chart/project-payment-chart.component.html:7]: I didn't try this (nor did I see this in the docs). I just copied what I had done in my prototype (whichwas based on the docs at the time).

File: services/121-service/src/payments/payments.service.ts [hunk 3]
```diff
@@ -540,23 +540,7 @@ export class PaymentsService {
...
   ): Promise<ProgramPaymentsStatusDto> {
-    if (payment) {
-      // TODO: refactor this once the "transaction/payment" entity exists,
-      // so that we can just get the status for a specific entity
-      const payments = await this.getPayments(programId);
-      const latestPayment = Math.max(...payments.map((p) => p.payment));
-      const isLatestPayment = payment === latestPayment;
-
-      if (!isLatestPayment) {
-        // only the latest payment can be in progress
```
Comment (aberonni) [inline services/121-service/src/payments/payments.service.ts:553]: FYI I removed this endpoint after realising, with Jannis, that this statement is actually incorrect.

Comment (elwinschmitz) [reply services/121-service/src/payments/payments.service.ts:553]: I think this wasn't used in the old Portal, right?

**Review by elwinschmitz — APPROVED** [review]
Assuming the 121-service change isn't a breaking change for the existing Portal: ✅

File: services/121-service/src/payments/payments.service.ts (hunk 3 above)
Comment (aberonni) [reply services/121-service/src/payments/payments.service.ts:553]: Yup, I created it for Portalicious in a separate PR.

**Pull request merged**